hpat/datatypes/hpat_pandas_series_functions.py

```diff
@@ -33,8 +33,8 @@
 
 import operator
 import pandas
+import numpy as np
```

Comment:
please do not rename numpy

Reply:
This is the the standard way of importing numpy...